hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/async/impl/NMClientAsyncImpl.java

```diff
@@ -433,7 +433,7 @@ public void getContainerStatusAsync(ContainerId containerId, NodeId nodeId) {
   }
 
   protected enum ContainerState {
-    PREP, FAILED, RUNNING, DONE,
+    PREP, FAILED, RUNNING, DONE
```

Comment:
out of curiosity, what harm is this comma causing?

Reply:
Thanks @ayushtkn  for the comment. I found that in most places in the Hadoop project, when enum is defined, the last element usually does not have any symbol, so I think it should be consistent here. This is my idea.
Welcome to continue to communicate.